File: app/src/main/java/com/attendance/contract/TeaStatContract.java
package com.attendance.contract;

import com.attendance.BasePresenter;
import com.attendance.BaseView;
import com.attendance.entities.ResultBean;

import java.util.List;

/**
 * Created by peiqin on 3/4/2017.
 */

public interface TeaStatContract {

    interface View extends BaseView<Presenter> {

        void showTip(String tip);

        void getStatSuccess(List<ResultBean> list);

    }

    interface Presenter extends BasePresenter {

        void getStatList();

    }

}

File: app/src/main/kotlin/com.attendance/contract/TeaStatContract.kt
package com.attendance.contract

import com.attendance.BasePresenter
import com.attendance.BaseView
import com.attendance.entities.ResultBean

/**
 * Created by peiqin on 3/4/2017.
 */

interface TeaStatContract {

    interface View : BaseView<Presenter> {

        fun showTip(tip: String)

        fun getStatSuccess(list: List<ResultBean>)

    }

    interface Presenter : BasePresenter {

        fun getStatList()

    }

}
